bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/communication/EvoCommunicator.java

```diff
@@ -217,7 +217,7 @@ private void createZoneTamperedFlags(ZoneStateFlags result, byte[] firstPage, by
     private void createZoneLowbatteryFlags(ZoneStateFlags result, byte[] firstPage, byte[] secondPage) {
         int pageOffset = panelType == PanelType.EVO48 ? 58 : 64;
         byte[] firstBlock = Arrays.copyOfRange(firstPage, 52, pageOffset);
-        if (panelType != PanelType.EVO192) {
+        if (!PanelType.isBigRamEvo(panelType)) {
             result.setZonesLowBattery(firstBlock);
```

Comment:
@jlaur In tag 3.2 of OH source, here we have a copy / paste error where we do a result.setZonesTampered(firstBlock) instead of result.setZonesLowBattery(firstBlock); 
That is most probably the issue with NPE in Evo48...

Probably I have fixed that in a rework in the 4.x branch where I have implemented the option to bypass a zone since I don't see it here.